script/state.py

```diff
@@ -15,6 +17,10 @@
 states_data = cur.fetchall()
 conn.close()
 
+if len(states_data)==0:
+    print("No data fetched from database ")
```

Comment:
use logging

Reply:
Already done . Please check the updated script.